packages/tasks/src/model-libraries.ts

```diff
@@ -208,6 +208,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		repoUrl: "https://github.com/Unbabel/COMET/",
 		countDownloads: `path:"hparams.yaml"`,
 	},
+	contexttab: {
+		prettyLabel: "ConTextTab",
+		repoName: "contexttab",
```

Comment:
```suggestion
		prettyLabel: "ConTextTab",
		repoName: "ConTextTab",
```

let's align naming (sorry, this is kinda a duplicate field)